docs/_layouts/default.html

```diff
@@ -9,7 +9,12 @@
     <meta name="description" content="{{ site.description }}">
 
     <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.baseurl }}">
-    <link rel="stylesheet" href="{{ "/css/default-theme.css" | prepend: site.baseurl}}" />
+    {% if site.env == 'development' %}
+      <link rel="stylesheet" href="http://localhost:8080/instantsearch.css" />
```

Comment:
As discussed, let's do an `npm run docs` that starts the correct npm, guard and jekyll commands so this is not an issue.